internal/executor/executor.go

```diff
@@ -85,7 +85,7 @@ func New(projectDir string, tasks []*api.Task, opts ...Option) (*Executor, error
 
 			// Lowest priority: task-specific
 			environment.NodeInfo(task.LocalGroupId, int64(len(tasks))),
-			environment.TaskInfo(task.Name, task.LocalGroupId),
+			environment.TaskInfo(task.Name),
```

Comment:
It was weird that it used `LocalGroupId`. Seems like a bug that was overwritten anyways after merging with other envs.